cmd/gpu-feature-discovery/main.go

```diff
@@ -158,10 +158,15 @@ func start(c *cli.Context, cfg *Config) error {
 		manager := resource.NewManager(config)
 		vgpul := vgpu.NewVGPULib(vgpu.NewNvidiaPCILib())
 
-		clientSets, err := cfg.kubeClientConfig.NewClientSets()
-		if err != nil {
-			return fmt.Errorf("failed to create clientsets: %w", err)
+		var clientSets flags.ClientSets
```

Comment:
just a note: NFD will soon (next release or so) make NodeFeature API enabled by default.

Reply:
What is the behaviour for a component such as GFD that may not use this API by default? Is the label file still honoured?